altiProfilAdmin/locales/en_US/admin.UTF-8.properties

```
@@ -18,4 +18,8 @@ form.altiProfileTable.label=Database table
 form.srid.label=SRID
 form.ignServiceKey.label=IGN key
 form.ignServiceUrl.label=IGN service URL
-form.altiresolution.label=Resolution
\ No newline at end of file
+form.altiresolution.label=Resolution
+form.connection_check.label=Connection status
+
+form.connection_check.ok=✓
+form.connection_check.error=Error while connecting to the table (see the log)
```

I think we can safely remove the translation of the emoji from `*.properties` ? :)
```suggestion
form.connection_check.error=Error while connecting to the table (see in the log panel)

```

Also can you set your IDE to add an empty line by default to each file ?
It's a good practice https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file (do a `cat` on the file)